### PR TITLE
[Main] Group module

### PR DESCRIPTION
**Pull request by colmazia**
Fixed: #group module

## Implementation details
- import group module in app.go
- fix proof and test

## Please ensure the following requirements are met before submitting a pull request:

- [x] The pull request is targeted against the correct target branch
- [ ] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [x] The pull request includes a description of the implementation/work done in detail.
- [x] The pull request includes any and all appropriate unit/integration tests
- [ ] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [x] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

File: scripts/start_yoda.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ yoda config chain-id bandchain
 yoda config validator $(bandd keys show validator -a --bech val --keyring-backend test)
 
 # setup execution endpoint
```
Comment (RogerKSI) [inline scripts/start_yoda.sh:11]: Please use mock url.

**Review by RogerKSI — CHANGES_REQUESTED** [review]

**Review by RogerKSI — APPROVED** [review]

**Review by RogerKSI — APPROVED** [review]

**Review by satawatnack — APPROVED** [review]

**Review by RogerKSI — APPROVED** [review]

File: client/grpc/oracle/proof/multi_store.go [hunk 2]
```diff
@@ -9,54 +9,55 @@ import (
...
-//	                                           ________________[AppHash]_________________
-//	                                          /                                          \
-//	                      _________________[I14]_________________                     __[I15]__
-//	                     /                                        \				 /          \
-//	          _______[I12]______                          _______[I13]________    [G]         [H]
-//	         /                  \                        /                    \
-//	    __[I8]__             __[I9]__                __[I10]__              __[I11]__
+//	                                           ________________[AppHash]_________________________
+//	                                          /                                                  \
+//	                      _________________[N15]_________________                             __[N16]___
```
Comment (taobun) [inline client/grpc/oracle/proof/multi_store.go:14]: Why need to change from I -> N

File: app/app.go [hunk 3]
```diff
@@ -56,6 +56,9 @@ import (
 	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
```
Comment (RogerKSI) [inline app/app.go:56]: Please help update the branch from `fix-rest-query` as there is some conflict.

**Review by RogerKSI — CHANGES_REQUESTED** [review]

File: client/grpc/oracle/proof/multi_store.go (hunk 2 above)
Comment (colmazia) [reply client/grpc/oracle/proof/multi_store.go:14]: because there is an 'I' used in the leaf node

**Review by RogerKSI — APPROVED** [review]

**Review by taobun — APPROVED** [review]

**Pull request merged**